LINK/usr/bin/evmserver.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 
-PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local:/usr/local/sbin:/usr/local/bin
```

Comment:
Yup, I tested this out on an appliance with a `systemctl restart evmserverd` so it checks out there.